src/display/editor/editor.js

```diff
@@ -492,6 +496,14 @@ class AnnotationEditor {
     this.div.scrollIntoView({ block: "nearest" });
   }
 
+  get _hasBeenMoved() {
+    return (
+      this.#initialPosition &&
```

Comment:
The getter should have a *consistent* return value:
```suggestion
      !!this.#initialPosition &&
```